src/js/containers/search/filters/location/LocationPickerContainer.jsx

```diff
@@ -3,14 +3,19 @@
  * Created by Kevin Li 10/30/17
```

Comment:
The only methods changed in this file is selectEntity, parseStates, and parseCountries. The rest are meaningless prettier changes (sorry).